### DISCUSSION TITLE
Why can't robots install knowledge from my knowledge base to answer questions?

### DISCUSSION THREAD

**Discussion by luke8sd** [FlowiseAI/Flowise#1789] [Q&A]
![企业微信截图_20240222163624](https://github.com/FlowiseAI/Flowise/assets/136670607/8e3c10af-71d8-45ce-b049-b478ae495542)
My qdrant contains information such as {\"question\" : \"How old is xiaoming?\", \"answer\" : \"He is 18.\"}. However, when I ask the robot How old is xiaoming?, most of the time it cannot answer according to the given answer, but only responds that it does not have this information.

**Comment by HenryHengZJ** [comment]
That is because your chain tool description needs to provide information on how and when LLM should use it. For example, use the tool when you need to search and answer something something

**Reply by luke8sd** [reply]
I tried to make him use this tool when asked questions or always use it, but he still seems unable to answer with the knowledge in the knowledge base. But when I remove the chain tool and its following parts, he can answer with the knowledge in the knowledge base. Could it be a conflict between two chatopenai?
